README.md

```diff
@@ -47,6 +47,13 @@ This provider's versions are able to install and manage the following versions o
 | Kubernetes 1.21 | ✓ | ✓ | ✓ |
 | Kubernetes 1.22 |   | ✓ | ✓ |
 
+<aside class="note warning">
+
+<h1> Warning </h1>
```

Comment:
nit: this is part of our README and it shows up weirdly big since github markdown doesn't support html warnings, it just prints it as a large header https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/b01e3ce99dd08ef7c7b15b97d4914414f92c91ea/README.md#-warning-

I wonder if changing this to h3 would fix it

Reply:
I'll try that

Reply:
![image](https://user-images.githubusercontent.com/3445370/150598144-6b37ddaf-eeff-46ed-8bf6-5f0634534be3.png)

Looks better in the README. Let's see how it renders in the book :crossed_fingers:

Reply:
it somewhat breaks the formatting of the warning box in the book https://deploy-preview-1992--kubernetes-sigs-cluster-api-provider-azure.netlify.app/

maybe we should get rid of the warning box format altogether since this page is meant to be the README of the repo and not a page in the book, it just happens to be embedded in the book

Reply:
I got fancy with the headers, and I think it works for both book and README. Let me know!